src/java.desktop/unix/native/libawt_wlawt/xdg-decoration-unstable-v1-client-protocol.h

```diff
@@ -0,0 +1,381 @@
+/* Generated by wayland-scanner 1.19.0 */
```

Comment:
Please use OpenJDK contribution header here

Reply:
The other generated files do not contain any such header AFAICT.

Reply:
I've added a header. I'm not sure if that's what you wanted though.

Reply:
Should be done.